ccpp/data/GFS_typedefs.F90

```diff
@@ -3737,6 +3761,62 @@ subroutine control_initialize (Model, nlunit, fn_nml, me, master, &
     Model%flag_for_scnv_generic_tend = .true.
     Model%flag_for_dcnv_generic_tend = .true.
 
+    Model%fh_dfi_radar     = fh_dfi_radar
```

Comment:
I wonder if that whole block 3764-3819 should go into its own subroutine.

Reply:
done